libs/motr/motr_core_k8s_lib.py

```diff
@@ -75,26 +76,26 @@ def _get_cluster_info(self):
             self.profile_fid = cluster_info["profiles"][0]["fid"]
             nodes_data = cluster_info["nodes"]
             for node in nodes_data:
-                if 'data' in node['name']:
+                if 'client' in node['name']:
                     nodename = node["name"]
                     self.cortx_node_list.append(nodename)
                     node_dict[nodename] = {}
-                    node_dict[nodename]['m0client'] = []
+                    node_dict[nodename]['motr_client'] = []
                     for svc in node["svcs"]:
                         if svc["name"] == "hax":
                             node_dict[nodename]['hax_fid'] = svc["fid"]
                             node_dict[nodename]['hax_ep'] = svc["ep"]
-                        if svc["name"] == "m0_client":
-                            node_dict[nodename]['m0client'].append({"ep": svc["ep"],
+                        if svc["name"] == "motr_client":
+                            node_dict[nodename]['motr_client'].append({"ep": svc["ep"],
                                                                     "fid": svc["fid"]})
         return node_dict
 
     def get_node_pod_dict(self):
         """
-        Returns all the node and data pod names in dict format
+        Returns all the node and motr client pod names in dict format
         """
         node_pod_dict = {}
-        cmd = "| awk '/cortx-data/ {print $1}'"
+        cmd = "| awk '/cortx-client/ {print $1}'"
```

Comment:
Is this the correct string to search?

Reply:
Yes. Previously motr client container was part of data pod now its part of client pod.